### PR TITLE
Add swagger

### PR DESCRIPTION
**Pull request by Terapool**
- Add 'rswag', 'rspec-rails' gems
- Initialize swagger (see '/api-doc' link)
- Give updates for README.md

### CODE REVIEW

**Review by Dmitryshl — APPROVED** [review]

File: app/controllers/categories_controller.rb [hunk 1]
```diff
@@ -5,9 +5,13 @@ class CategoriesController < ApplicationController
 
   def index
     @categories = Category.page(params[:page])
+
+    render json: @categories, only: %i[id name description]
```
Comment (fooooxmr) [inline app/controllers/categories_controller.rb:9]: Changes are not related to this PR. You can rewrite the whole controller, or remove changes at all. We have a separate ticket for this.

File: spec/requests/categories_spec.rb [hunk 2]
```diff
@@ -0,0 +1,95 @@
+require 'swagger_helper'
+
+RSpec.describe 'categories', type: :request do
+  path '/{locale}/categories' do
```
Comment (fooooxmr) [inline spec/requests/categories_spec.rb:4]: The same. Or remove this file, or correct all data inside.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: spec/requests/categories_spec.rb (hunk 2 above)
Comment (Terapool) [reply spec/requests/categories_spec.rb]: Done - the file is removed

File: app/controllers/categories_controller.rb (hunk 1 above)
Comment (Terapool) [reply app/controllers/categories_controller.rb]: Done - the file is removed

**Review by fooooxmr — APPROVED** [review]

**Review by ritamma — APPROVED** [review]
